prime-dotnet-webapi/Services/OrganizationService.cs

```diff
@@ -57,7 +57,8 @@ public async Task<IEnumerable<OrganizationSearchViewModel>> GetOrganizationsAsyn
                 .If(!string.IsNullOrWhiteSpace(searchOptions.TextSearch), q => q
                     .Search(
                         o => o.Name,
-                        o => o.DisplayId.ToString())
+                        o => o.DisplayId.ToString(),
+                        o => o.SigningAuthority.FirstName + " " + o.SigningAuthority.LastName)
```

Comment:
Need to also update `OrganizationListViewModel.MatchedOn()`

Reply:
Updated, please check, since we're checking both fields combined I added `SigningAuthority.FirstName` and `SigningAuthority.LastName` together using the same logic